File: problems/easy/savings/savings.py
# Simple Interest Calculation
def simple_interest(principal, rate, time):
    interest = (principal * rate * time) / 100
    total_amount = principal + interest
    return total_amount

# Input data
principal = float(input("Enter the principal amount: "))
rate = float(input("Enter the annual interest rate (in percentage): "))
time = float(input("Enter the time (in years): "))

# Calculate total amount
total = simple_interest(principal, rate, time)

# Output result
print(f"The total amount after {time} years is: {total:.2f}")

File: problems/easy/savings/savings.cpp
#include <iostream>
using namespace std;

int main() {
    double principal, rate, time, interest, total;

    // Input the principal, interest rate, and time
    cout << "Enter the principal amount: ";
    cin >> principal;
    cout << "Enter the annual interest rate (in percentage): ";
    cin >> rate;
    cout << "Enter the time (in years): ";
    cin >> time;

    // Calculate the simple interest
    interest = (principal * rate * time) / 100;

    // Calculate the total amount
    total = principal + interest;

    // Output the result
    cout << "The total amount after " << time << " years is: " << total << endl;

    return 0;
}
